pipit/readers/projections_reader.py

```diff
@@ -555,18 +555,18 @@ def _read_log_file(self, pe_num: int) -> pandas.DataFrame:
                 details = {
                     "From PE": pe,
                     "Message Type": mtype,
-                    "Entry Name": sts_reader.get_entry_name(entry),
+                    "Entry Name": "Processing",
                     "Event ID": event,
                     "Message Length": msglen,
                     "CPU End Time": cpu_end_time,
                     "perf counts list": perf_counts,
                 }
 
-                data["Name"].append("Processing")
+                data["Name"].append(sts_reader.get_entry_name(entry))
                 data["Event Type"].append("Leave")
                 data["Timestamp (ns)"].append(time)
                 data["Process"].append(pe_num)
-                data["Details"].append(sts_reader.get_entry_name(entry))
+                data["Details"].append(details)
```

Comment:
Are we adding details both in Enter and Leave now as opposed only in Enter events?